docs/src/idiomatic/assignments.md

```diff
@@ -0,0 +1,129 @@
+# Idiom 1: Update state variables with assignments
+For each TLA+ variable `x` defined in a specification, Apalache requires that every possible  transition defines an assignment to `x'`.
+
+## Assignments and Assignment Candidates
+An _assignment candidate_ for `x` is an expression that has one of the following forms:
+- `x' = e`, 
+- `x' \in e`,
+- `UNCHANGED x`, or
+- `x' := e`
+
+While a transition may contain multiple assignment candidates for the same variable, not all of them are chosen as _assignments_ by Apalache. The sections below describe how the assignments are selected.
+
+### Minimality
+Assignments aren't spurious; each variable must have at least one assignment per transition operator, but no more than necessary to satisfy all of the additional constraints below.
+
+If all possible transitions fail to assign one or more variables, an exception of the following shape is thrown:
+```
+Assignment error: No assignments found for: x, z 
+```
+
+Such exceptions are usually the result of adding a `VARIABLE` without any accompanying TLA+ code relating to it. 
+The case where at least one transition, but not all of them, fails to assign a variable is shown below.
+
+### Syntax Order
+For the purpose of evaluating assignments, Apalache considers the left-to-right syntax order of conjuncts. 
```

Comment:
This is not a friendly language for TLA+ users. Can we say "and (/\\)" instead of conjunct? The point of our manual is not to teach them first-order logic, but to explain what we are doing.

Reply:
I think that's hard, you're not talking about `/\`, but about the sub-formulas that make up a `/\` (for which a name already exists and is used here - conjunct).

Reply:
I think users should be trusted to look up words they don't know, and that we should use the right language for what we're talking about.

But, if we don't think users can be trusted to google words they don't know, then perhaps we can add something along the lines of one of the two following fixes to address Igor's point?

```suggestion
For the purpose of evaluating assignments, Apalache considers the left-to-right syntax order of [conjuncts](https://en.wikipedia.org/wiki/Logical_conjunction). 
```

or

```suggestion
For the purpose of evaluating assignments, Apalache considers the left-to-right syntax order of conjuncts (the terms conjoined in a conjunction, e.g., `a` and `b` in `a /\ b`). 
```